isaaclab.sh

```diff
@@ -90,8 +90,14 @@ extract_isaacsim_exe() {
     local isaacsim_exe=${isaac_path}/isaac-sim.sh
     # check if there is a python path available
     if [ ! -f "${isaacsim_exe}" ]; then
-        echo "[ERROR] No Isaac Sim executable found at path: ${isaacsim_exe}" >&2
-        exit 1
+        # check for installation using Isaac Sim pip
+        if [ $(python -m pip list | grep -c 'isaacsim-rl') -gt 0 ]; then
+            # Isaac Sim - Python packages entry point
+            local isaacsim_exe="isaacsim omni.isaac.sim"
```

Comment:
```suggestion
            local isaacsim_exe="isaacsim omni.isaac.sim.kit"
```

Reply:
There is no need to add `.kit`
https://docs.omniverse.nvidia.com/isaacsim/latest/installation/install_python.html#launching-isaac-sim-experiences

Reply:
Also, in next Isaac Sim release the `omni.isaac.sim.kit` experience file will be the default. So `isaacsim` will launch full Isaac Sim